packages/solana/lib/solana.dart

```diff
@@ -8,7 +8,7 @@ export 'src/spl_token/associated_account.dart';
 export 'src/spl_token/spl_token.dart';
 export 'src/system_program/system_program.dart';
 export 'src/token_program/token_program.dart';
-export 'src/utils.dart';
+export 'src/utilities.dart';
```

Comment:
I do not remember why I did this, it was a few days ago and now I don't remember. I can rename it the way it was and if something makes me rename it again I will give an explanaiton, otherwise we can keep the original name.